### ISSUE TITLE
[Feature Request] Specify dependency by tag

### ISSUE THREAD

**Issue by soichisumi** [stormcat24/protodep#28]
I want to specify dependency by tag, but protodep seems not to specify dependency with tag.

ex: protodep.toml
```toml
proto_outdir = "./proto"

[[dependencies]]
  target = "github.com/stormcat24/protodep/protobuf"
  revision = "0.0.3"
```

result:
```
> protodep up -f

[INFO] force update = true
[INFO] identity file = id_rsa
[INFO] use SSH protocol
[INFO] Getting github.com/stormcat24/protodep
Error: get commit is failed: object not found
Usage:
  protodep up [flags]

Flags:
  -f, --force                  update locked file and .proto vendors
  -h, --help                   help for up
  -i, --identity-file string   set the identity file for SSH (default "id_rsa")
  -p, --password string        set the password for SSH

get commit is failed: object not found
```

Is there any way to specify dependencies by tag?

**Comment by stormcat24** [comment]
@soichisumi Thanks for using protodep.
This issue is fixed in [0.0.6](https://github.com/stormcat24/protodep/releases/tag/0.0.6).
Please check it 🙏

**Comment by soichisumi** [comment]
[0.0.6](https://github.com/stormcat24/protodep/releases/tag/0.0.6) release worked perfectly. 
Thank you!
